src/CacheManager.Microsoft.Extensions.Caching.Memory/MemoryCacheHandle`1.cs

```diff
@@ -152,10 +173,7 @@ protected override bool AddInternalPrepared(CacheItem<TCacheValue> item)
             var options = GetOptions(item);
             _cache.Set(key, item, options);
 
-            if (item.Region != null)
-            {
-                _cache.RegisterChild(item.Region, key);
-            }
+            _cache.RegisterChild(item.Region ?? GetType().Name, key);
```

Comment:
Yeah I think that would be fine, I miss interpreted the code, you just use the GetType of the cache handle itself.
But again, you also have to remove keys from that "standard" region, if a key gets removed ;)